packages/client/components/ErrorBoundary.tsx

```diff
@@ -15,20 +17,31 @@ interface State {
   error?: Error
   errorInfo?: ErrorInfo
   eventId?: string
+  isOldBrowserErr: boolean
 }
 
 class ErrorBoundary extends Component<Props, State> {
   state: State = {
     error: undefined,
     errorInfo: undefined,
-    eventId: undefined
+    eventId: undefined,
+    isOldBrowserErr: false
+  }
+
+  componentDidUpdate() {
+    const {error, isOldBrowserErr} = this.state
+    if (!error || isOldBrowserErr) return
+    const {atmosphere} = this.props
+    const {viewerId} = atmosphere
+    SendClientSegmentEventMutation(atmosphere, 'Fatal Error', {viewerId})
```

Comment:
I don't get why you couldn't do this in `componentDidCatch`?

Reply:
`componentDidCatch` re-renders four times and would send four events to Segment